pyproject.toml

```diff
@@ -29,14 +29,15 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
 ]
 dynamic = ["version"]
+
 dependencies = [
     "colorama>=0.3.9",
     "configobj>=5.0.6",
     "distro>=1.3",
     "dpath<3,>=2.1.0",
     "dvc-data>=2.22.0,<2.23.0",
     "dvc-http>=2.29.0",
-    "dvc-render>=0.3.1,<1",
+    "dvc-render@git+https://github.com/iterative/dvc-render@refs/pull/142/head",
```

Comment:
[I] Make sure new version of `dvc-render` has been released and this has been updated.